evalml/tests/automl_tests/test_automl.py

```diff
@@ -1268,7 +1272,7 @@ class Pipeline2(DummyPipeline):
     elif problem_type_value == ProblemTypes.TIME_SERIES_REGRESSION:
         automl = AutoMLSearch(X_train=X, y_train=y, problem_type=problem_type_value, max_iterations=3,
                               allowed_pipelines=[Pipeline1, Pipeline2], objective=objective,
-                              additional_objectives=[], problem_configuration={'gap': 0, 'max_delay': 0}, n_jobs=1)
+                              additional_objectives=[], problem_configuration={'gap': 0, 'max_delay': 0}, train_best_pipeline=False, n_jobs=1)
```

Comment:
Mocking fit for time series regression resulted in the following error when trying to fit the best pipeline during training: `AttributeError: 'NoneType' object has no attribute 'fit'`. The pipeline didn't have a `self._component_obj` value. Would really appreciate input on why it wouldn't work for only time series regression

Reply:
Was the best pipelines the baseline pipeline? The baseline estimator does not have a [component object ](https://github.com/alteryx/evalml/blob/main/evalml/pipelines/components/estimators/regressors/time_series_baseline_estimator.py#L45)

Reply:
Interestingly enough, when the best_pipeline is the 'Time Series Baseline Estimator', the tests pass, but when it's `Pipeline1` or `Pipeline2`, with the estimators being the 'MockTimeSeriesRegressor', they fail. That mock also sets `component_obj` to None, which might be why it's failing.